client/src/main/webjar/map/map.component.js

```diff
@@ -16,6 +16,31 @@
         var map = this;
 
         /**
+         * Returns the current time in milliseconds so we can bust google's caching of our generated KML files.
+         * @returns {Number} current time in milliseconds.
+         */
+        function getMsTime() {
+            return Date.now();
+        }
+
+        /**
+         * Returns the program IDs that the user is interested in as a comma delimited string.
+         * @returns {String} comma delimited string of the IDS of the programs to display.
+         */
+        function getProgramIdsString() {
+            return "1,2";
```

Comment:
I made this something hard-coded that will demo for now.  Jay and I will work out the details of our interaction later (what the client passes in on the search and what information the server must embed into the URL and how the server sends that back to the client).